tokens/contracts/erc-1155/ERC1155Base.sol

```diff
@@ -18,6 +18,35 @@ abstract contract ERC1155Base is OwnableUpgradeable, ERC1155DefaultApproval, ERC
         return ERC1155DefaultApproval.isApprovedForAll(_owner, _operator);
     }
 
+    function supportsInterface(bytes4 interfaceId) public view virtual override(ERC1155Lazy, ERC165Upgradeable) returns (bool) {
+        return super.supportsInterface(interfaceId);
+    }
+
+    function burn(address account, uint256 id, uint256 value) public virtual override {
+        uint256 burnMinted = _burnLazy(account, id, value);
```

Comment:
Is it good name for variable?
What does it mean?
Is it "left to burn"?

Reply:
change to leftToBurn